nuxt.config.ts

```diff
@@ -18,8 +17,13 @@ export default defineNuxtConfig({
   ],
   nitro: {
     prerender: {
-      crawlLinks: false
-    }
+      crawlLinks: true,
+      routes: ['/en/', '/nl/'],
+    },
+  },
+  routeRules: {
+    '/**': { swr: true },
```

Comment:
Would want to use `static: true` but doesn't work for some reason...

Reply:
Does this mean that every route will now be swr rendered? Or just the routes that didn't get prerendered by the crawler?

Reply:
Just the routes that didn't get prerendered, I actually see [the docs](https://nuxt.com/docs/guide/concepts/rendering#hybrid-rendering) have changed in the meantime adding isr:
> isr - The behavior is the same as swr except that we are able to add the response to the CDN cache on platforms that support this (currently Netlify or Vercel)